packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -189,6 +190,9 @@ export const HelpCenterContactForm = ( props: HelpCenterContactFormProps ) => {
 		( event: React.MouseEvent< HTMLAnchorElement, MouseEvent >, result: SearchResult ) => {
 			event.preventDefault();
 
+			const currentRoute = location.pathname + location.search;
+			const navigatingFromGPTResponse = currentRoute === '/contact-form?mode=FORUM&show-gpt=true';
```

Comment:
We already have `mode` and `showingGPTResponse`. Maybe the check would be better if we base it on that